### PR TITLE
Add Feature coding standards

### PR DESCRIPTION
**Pull request by Arakhaa**
Can you take a look at the work of Etienne and I?

### CODE REVIEW

**Comment by valotg** [comment]
Ok for me, I guess we are ready to merge.

**Review by valotg — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -139,6 +190,40 @@ Here is a non-exhaustive list of Symfony conventions
...
 
 ---
+### Comments Standards
+
+**When should we comment?**
+When the need or need comfort required so that other group members can understand easily function and functionality.
+
+Remember that the comments are portions of the source code disregarded by the compiler or interpreter, because they are not necessary for the program to run. They are usually inserted in the code so that it is easy to understand and that it can be changed easily in the  future.
+
+Developer tends to underestimate the benefits of commenting, but comment **!=** document.
```
Comment (4kh) [inline README.md:200]: maybe documentation would be more correct to say

Comment (Arakhaa) [reply README.md:200]: Here, "document" means "documenter"

**Review by 4kh — APPROVED** [review]
Seems ok to me

**Pull request merged**